src/components/ExplorePage/index.js

```diff
@@ -0,0 +1,118 @@
+import { Hidden } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import dynamic from "next/dynamic";
+import PropTypes from "prop-types";
+import React, { useEffect, useState } from "react";
+import useSWR from "swr";
+
+import fetcher from "@/pesayetu/utils/fetcher";
```

Comment:
I hear you and SWR use the same name but `fetcher` is weird to me...`fetchJson`, `fetchAsJson`, etc., sound better.